cves/kernel/CVE-2017-18079.yml

```diff
@@ -225,7 +235,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: drivers
   note:
```

Comment:
Maybe add a short note about how you found out it's in the drivers subsystem

Reply:
Sure, I'll add that.